File: Consecutive Characters.py
class Solution:
    def maxArea(self, h: int, w: int, a: List[int], b: List[int]) -> int:
        a=[0]+a+[h]
        b=[0]+b+[w]
        a.sort()
        b.sort()
        mh=mv=0
        for h in range(len(a)-1):
            mh=max(mh, a[h+1]-a[h])
        for h in range(len(b)-1):
            mv=max(mv, b[h+1]-b[h])
        m=10e+8+7
        return int((mh*mv)%m)
        

File: Consecutive Characters.cpp
class Solution
{
public:
    int maxPower(const string &s)
    {
        char ch = s[0];
        int res = 0, count = 1;
        for (int i = 1; i < s.size(); ++i)
        {
            if (s[i] != ch)
            {
                if (res < count)
                    res = count;
                ch = s[i];
                count = 1;
            }
            else
                ++count;
        }
        return max(res, count);
    }
};